src/middleware/not-found.ts

```diff
@@ -0,0 +1,5 @@
+import { Request, Response } from "express";
+
+export const notFoundMiddleware = (req: Request, res: Response): void => {
```

Comment:
I would prefer to put this in the `app.ts` where it's declared. It's a one-liner that I shouldn't have to search for to find, I think it would make more sense to in-line the handler function with `app.use`

Reply:
fixed